ci-operator/step-registry/gather/proxy/gather-proxy-commands.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+if test ! -f "${SHARED_DIR}/proxyregion"
+then
+	echo "No proxyregion, so unknown AWS region, so unable to tear down ."
+	exit 0
+fi
+
+REGION="$(cat "${SHARED_DIR}/proxyregion")"
+PROXY_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
+
+# cleaning up after ourselves
+aws --region "${REGION}" s3 rb "s3://${PROXY_NAME}" --force
+
+STACK_NAME="${PROXY_NAME}-proxy"
+
+if test -f "${SHARED_DIR}/proxy-instance-ids.txt"
+then
+  cat "${SHARED_DIR}/proxy-instance-ids.txt" | sort | uniq | while read -r INSTANCE_ID
+  do
+	  echo "Gathering console logs for ${INSTANCE_ID}"
+	  aws --region "${REGION}" ec2 get-console-output --instance-id "${INSTANCE_ID}" --output text > "${ARTIFACT_DIR}/${INSTANCE_ID}" &
```

Comment:
You shouldn't need to do this directly, you should be able to append the proxy instance ID to [`${SHARED_DIR}/aws-instance-ids.txt`][1] and let the `gather-aws-console` step gather them for you.

[1]: https://github.com/openshift/release/blob/73a6f4db87b209ece25fd6b3fd094d58df2aaff3/ci-operator/step-registry/gather/aws-console/gather-aws-console-ref.yaml#L17

Reply:
the issue i noticed was if for some reason the cluster wouldn't come up, we couldn't grab the instance logs for the proxy due to the checks done in `gather-aws-console`